todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -175,4 +177,32 @@ public DayTodoReadyResponseDto updateIsReady(@PathVariable UUID dayId, @PathVari
             throw new DayTodoReadyCanNotBeUpdatedException("Day todo ready can't be updated.", ex);
         }
     }
+
+    @PatchMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "Change orderNumber id existed Todo", response = DayTodoResponseDto.class, responseContainer = "List")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Created"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+    public List<DayTodoResponseDto> changeOrderNumberDayTodo(@PathVariable Map<UUID,Integer> map ) {
+        try {
+            Map<UUID,Integer> mapExistDayTodo = map.entrySet().stream()
+                    .filter(entry -> dayTodoService.isExistDayTodo(entry.getKey()))
+                    .collect(Collectors.toMap(entry -> entry.getKey(), entry -> entry.getValue()));
+
+            List<DayTodoVO> dayTodoVOS = mapExistDayTodo.entrySet().stream()
+                    .map(entry ->dayTodoService.updateDayTodoById(entry.getKey(),entry.getValue()))
+                    .collect(Collectors.toList());
+
+            return dayTodoVOS.stream().map(dayTodoVO -> dayTodoRestMapper.toDayTodoResponseDto(dayTodoVO))
+                    .collect(Collectors.toList());
```

Comment:
Too much logic in the Controller, you should accept and validate data in a controller but all business logic and mappings should be done in Service. Maybe now it's not like this in other Controllers but we should make it like this.